app/models/agents/aftership_agent.rb

```diff
@@ -0,0 +1,156 @@
+require 'uri'
+
+module Agents
+  class AftershipAgent < Agent
+
+    default_schedule "every_10m"
+
+    description <<-MD
+      The Aftership agent allows you to track your shipment from aftership and emit them into events.
+
+      To be able to use the Aftership API, you need to generate an `API Key`. You need a paying plan to use their tracking feature.
+
+      You can use this agent to retrieve tracking data. You have to provide a specific `path` request and its associated option.
+ 
+      To get all trackings for your packages please enter `path` for key and `trackings` for the option.
+      To get tracking for a specific tracking number, add the extra keys `slug`, `tracking_number` and their associated values. Set `single_tracking_request` to true.
+
+      To get the last checkpoint of a package set key to `path` and option to `last_checkpoint`. Please provide `slug` and `tracking_number`. Set `last_checkpoint_request` to true.
+
+      `slug` is a unique courier code. 
+
+      You can get a list of courier information here `https://www.aftership.com/courier`
+
+      Required Options:
+
+      * `api_key` - YOUR_API_KEY.
+      * `path and its associated options`
+    MD
+
+    event_description <<-MD
+      A typical tracking event has 3 objects (attributes, tracking, and checkpoint) and it looks like this
+    {
```

Comment:
Its rendered correctly now but I removed the attributes object because its not returning anything useful. Compared to other agents, its a bit on the long side. Hope this works out.